dev3/2024-07-02/example-metadata/min-specimen-biosample.json

```diff
@@ -0,0 +1,74 @@
+{
+    "@context": [
+        "https://w3id.org/ro/crate/1.1/context",
+        {
+            "organism_classification": "https://schema.org/taxonomicRange",
+            "BioChemEntity": "https://schema.org/BioChemEntity",
+            "obo": "http://purl.obolibrary.org/obo/",
+            "acquisiton_method": {
```

Comment:
As a sidenote, the ARC community (https://github.com/nfdi4plants/ARC-Symposium, etc.) has suggested that the new term `LabProcess` be used: https://github.com/BioSchemas/specifications/pull/669 In the graph that would then add an entity which takes inputs and outputs which may touch on some of what is happening here with the `@reverse` of `result`.